scripts/tests

```diff
@@ -81,7 +83,7 @@ def base_run(bin, force_qemu, verbose, extra_make_envs={}, extra_make_args=[]):
             sys.exit(1)
 
         result = subprocess.run(
-            [f"./{bin}"],
```

Comment:
Nit: This should ideally print a short message to the user explaining what they're being asked `sudo` rights for.